modules/platforms/cpp/odbc-test/src/api_robustness_test.cpp

```diff
@@ -795,6 +795,31 @@ BOOST_AUTO_TEST_CASE(TestSQLSetStmtAttr)
     SQLSetStmtAttr(stmt, SQL_ATTR_ROW_ARRAY_SIZE, 0, 0);
 }
 
+BOOST_AUTO_TEST_CASE(TestSQLSetStmtAttrGetStmtAttr)
+{
+    // check that statement array size is set correctly 
+
+    Connect("DRIVER={Apache Ignite};address=127.0.0.1:11110;schema=cache");
+
+    SQLCHAR buffer[ODBC_BUFFER_SIZE];
+    SQLINTEGER resLen = 0;
+
+    // repeat test for different values
+    SQLULEN valList[5] = {10, 52, 81, 103, 304};
+    for (SQLULEN val : valList) {
```

Comment:
Sorry, we do not use C++11 yet. Can you change this part to index form of for loop (`for (i = 0; ...)`)?

Reply:
no worries. I made the change